tutorials/tutorial7/tutorial_7.rb

```diff
@@ -0,0 +1,48 @@
+# person parent class
+class Person
+  attr_accessor :name
+
+  def initialize(name)
+    @p_name = name
+  end
+# print_name method
+  def print_name
+    puts "Name : #{@p_name}"
+  end
+end
+# student child class
+class Student < Person
+  attr_accessor :roll_no
+
+  def initialize(name, roll_no)
+    @p_name = name
+    @stu_rn = roll_no
```

Comment:
Please give variables as easy to read.
@stu_rn => @student_roll or something.....

Reply:
Thanks for correcting me bro @HtetTunLu .I updated the tutorial_7.rb on [a03283a](https://github.com/maythuzarlinn/Ruby-Training/pull/7/commits/a03283acd59ac8fd6a46b659941e2a32af0c10f5) according to your instruction.